tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Requests/CompleteOrderRequestTests.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using NHSD.BuyingCatalogue.Ordering.Api.Services.CompleteOrder;
+using NUnit.Framework;
+
+namespace NHSD.BuyingCatalogue.Ordering.Api.UnitTests.Requests
+{
+    [TestFixture]
+    [Parallelizable(ParallelScope.All)]
+    internal sealed class CompleteOrderRequestTests
+    {
+        [Test]
+        public void Constructor_NullOrder_ThrowsArgumentNullException()
```

Comment:
A test for the happy path is probably worth adding too, making sure .Order is what's passed in